arch/arm/src/sama5/sam_gmac.c

```diff
@@ -1,5 +1,5 @@
 /****************************************************************************
- * arch/arm/src/sama5/sam_gmac.c
+ * arch/arm/src/chip/sam_gmac.c
```

Comment:
arch/arm/src/chip/ is the simlink of arch/aarm/src/sama5
```suggestion
 * arch/arm/src/sama5/sam_gmac.c
```